src/FEValues/GeometryMapping.jl

```diff
@@ -121,15 +121,32 @@ function otimes_returntype(#=typeof(x)=#::Type{<:Vec{dim,Tx}}, #=typeof(d2Mdξ2)
     return Tensor{3,dim,promote_type(Tx,TM)}
 end
 
-@inline function calculate_mapping(::GeometryMapping{0}, q_point, x)
+@inline function calculate_mapping(gip::VectorizedInterpolation, ξ::Vec, cell_coordinates::AbstractVector{<:Vec})
+    return calculate_mapping(gip.ip, ξ, x)
+end
+
+@inline function calculate_mapping(gip::ScalarInterpolation, ξ::Vec, cell_coordinates::AbstractVector{<:Vec})
+    n_basefuncs = getnbasefunctions(gip)
+    @boundscheck checkbounds(cell_coordinates, Base.OneTo(n_basefuncs))
+
+    fecv_J = zero(otimes_returntype(Vec{sdim,T}, Vec{rdim,T}))
+    @inbounds for j in 1:n_basefuncs
+        dMdξ = shape_gradient(gip, ξ, j)
+        #fecv_J += cell_coordinates[j] ⊗ dMdξ[j]
+        fecv_J += otimes_helper(cell_coordinates[j], dMdξ[j])
+    end
+    return MappingValues(fecv_J, nothing)
+end
+
```

Comment:
Separate PR? 
Doesn't seem used in this PR?

Reply:
https://github.com/Ferrite-FEM/Ferrite.jl/pull/1018